### ISSUE TITLE
Multitask training problem

### ISSUE THREAD

**Issue by PasqualeAuriemma** [keras-team/keras#10162]
Hi everyone, i have a model with three output,  one for object class, one for attribute class and one for action class. I would to select one of three tasks inside each iteration and train the model with the optimization of selected layer loss.
Is there a way to ignore untrained layers?

**Comment by KDercksen** [comment]
You could write separate loss functions for each layer. Within those loss functions, select the proper predictions from `y_true` and `y_pred` and calculate loss value with that.

**Comment by PasqualeAuriemma** [comment]
Thanks for your help.
